types/autosuggest-highlight/match/index.d.ts

```diff
@@ -0,0 +1 @@
+export default function match(text: string, query: string): number[] | number[][];
```

Comment:
Because [these are both defined as `module.export = `-style declarations](https://github.com/moroshko/autosuggest-highlight/blob/7bb02bd439cddbd074e30cfc2c3e5d75905ee292/src/match.js#L15), it would be more appropriate to define these as

```ts
function match(text: string, query: string): number[] | number[][];
export = match;
```